portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -90,4 +93,23 @@ private String convertToJson(List<HashMap<String, String>> list) {
     String json = gson.toJson(list);
     return json;
   }
+
+  /**
+   * Converts queryString into HashMap of field and value pairs obtained 
+   * from the queryString
+   * @param queryString query string of which the field value pair are to be processed
+   * @return HashMap of field:value mappings
+   */
+  private HashMap<String, String> getFieldValues(String queryString) {
```

Comment:
Have you looked at something like http://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/client/utils/URLEncodedUtils.html#parse(org.apache.http.util.CharArrayBuffer,%20java.nio.charset.Charset,%20char...) ?

Reply:
For some reason, it's saying that the "package org.apache.http.client.utils does not exist" when I import the URLEncodedUtils class using `import org.apache.http.client.utils.URLEncodedUtils`. I have done some Googling but without much luck so far. I'll leave the code I have right now so that I can move on to the next step, but I'll ask in tech support.